.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md 
```

Comment:
@Tufahel Hello Tufahel. Good Morning. I have one question. You said my test.md should be ignored and shouldn't appear in my github repository. but how can i do it? because i dont know how. if you can help me would be great!. Thank you for the review.

Reply:
@Joaquin123455, I suggest trying to follow the way.
  - First, delete `test.md` and `.gitignore` files from your local device(laptop/desktop).
  - Then create `test.md` file and write something as per project requirement.
  - Then create `.gitignore` and write `node_modules` and `test.md` they same way you wrote before.
  - Now, before push, you should check your ide:(such as vscsode/atom), you can see `.test.md` and `node_moudules` will be the same color if they are ignored perfectly.
  - Now you can push.

Happy Coding. 😄